src/main/java/ab/common/lib/register/RecipeListAB.java

```diff
@@ -486,8 +490,9 @@ public static void setupCrafting() {
         // Sphere of Attraction recipe
         GameRegistry.addShapedRecipe(
                 new ItemStack(ItemListAB.itemAntigravityCharm),
-                new Object[] { " G ", "GDG", "ERE", Character.valueOf('E'), new ItemStack(ModItems.manaResource, 1, 7),
-                        Character.valueOf('G'), new ItemStack(ModBlocks.elfGlass), Character.valueOf('D'),
+                new Object[] { " G ", "GDG", "ERE", Character.valueOf('E'),
+                        OreDict.preference("plateElvenElementium", OreDict.ELEMENTIUM_PLATE), Character.valueOf('G'),
+                        new ItemStack(ModBlocks.elfGlass), Character.valueOf('D'),
```

Comment:
Points from the previous review that have not been addressed:
1: Use oredictionary recipes instead of basic recipes (look up `ShapedOreRecipe`) so that the ore dictionary is actually used as intended instead of pretending.
2: remove `Object[] { ... }`
3: remove references to `Character.valueOf`
4: remove anything that pins an oredictionary reference to a single item (`OreDict.preference(...)`, `getOres(...).get(0)`) and use the string directly in the recipe
5: make sure the recipe letter and the matching ingredient are on the same line of code.